src/cf-modals/src/Modal.js

```diff
@@ -0,0 +1,172 @@
+/* ==========================================================================
+   Modal Organism
+   ========================================================================== */
+
+const UNDEFINED = require( 'cf-atomic-component/src/utilities/config' ).UNDEFINED;
```

Comment:
I don't see why [this is fine](https://github.com/cfpb/capital-framework/blob/canary/src/cf-expandables/src/Expandable.js#L18), but it's not fine here.